.github/workflows/build_deploy_and_test.yml

```diff
@@ -0,0 +1,314 @@
+              "https://api.github.com/repos/${CI_PROJECT_NAME}/commits/${COMMIT_SHA}/pulls" | \
+              python3 -c "import sys, json; print(json.load(sys.stdin)[0]['html_url'])"`
+          else
+            export CI_BRANCH="main"
+          fi
+          export CI_PROJECT_REPONAME=`curl -u "${GITHUB_OWNER}:${GITHUB_TOKEN}" \
+            -H "Accept: application/vnd.github.groot-preview+json" \
+            "https://api.github.com/repos/${CI_PROJECT_NAME}/commits/${COMMIT_SHA}/pulls" | \
+            python3 -c "import sys, json; print(json.load(sys.stdin)[0]['base']['repo']['name'])"`
+          export CI_PROJECT_USERNAME=$GITHUB_REPOSITORY_OWNER
+          /build-tools-ci/scripts/set-environment
+          GITHUB_WORKFLOW_URL=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
+          echo "export CI_BUILD_URL='${GITHUB_WORKFLOW_URL}'" >> $BASH_ENV
+          echo "export CI_NODE_INDEX=0" >> $BASH_ENV
+          echo "export CI_REPOSITORY_URL='https://github.com/${GITHUB_REPOSITORY}'" >> $BASH_ENV
+          echo "export ARTIFACTS_DIR_URL='${GITHUB_WORKFLOW_URL}/#artifacts'" >> $BASH_ENV
+
+  build_php:
+    container:
+      image: quay.io/pantheon-public/build-tools-ci:6.x
+      options: --user root
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache composer cache
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-composer-cache
+        with:
+          path: ~/.composer/cache
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Cache vendor folder
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-vendor
+        with:
+          path: ./vendor
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('composer.lock') }}
+
+      - name: Cache web folder
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-web
+        with:
+          path: ./web
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('composer.lock') }}
+
+      - name: Cache drush folder
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-drush
+        with:
+          path: ./drush
+          key: ${{ runner.os }}-build-${{ env.cache-name }}
+
+      # Set TERMINUS_ENV and related environment variables.
+      # https://github.com/pantheon-systems/docker-build-tools-ci/blob/6.x/scripts/set-environment
+      - name: setup-environment-vars
+        run: /build-tools-ci/scripts/set-environment
+
+      - name: run PHP build step
+        run: ./.github/workflows/php
+
+  code_sniff_unit_test:
+    runs-on: ubuntu-latest
+    container:
+      image: drupaldocker/php:7.2-cli
```

Comment:
Is it possible to use a PHP version thats more current?

Reply:
I swapped in a different container